CRM/Import/Parser.php

```diff
@@ -2352,33 +2352,37 @@ protected function getDedupeFields(string $contactType): array {
   protected function getAllContactFields(string $prefix = 'Contact.'): array {
     $allContactFields = (array) Contact::getFields()
       ->addWhere('readonly', '=', FALSE)
-      ->addWhere('type', 'IN', ['Field', 'Custom'])
+      ->addWhere('usage', 'CONTAINS', 'import')
       ->addWhere('fk_entity', 'IS EMPTY')
+      ->setAction('save')
```

Comment:
FYI I'm pretty sure the psudeo-action "save" just maps to "create"

Reply:
@colemanw well it feels more accurate than saying 'create'